scripts/fires/perimeter/wfigs_fire_perimeter.py

```diff
@@ -24,12 +24,27 @@
 _FIRE_DCID_FORMAT = "fire/irwinId/{id}"
 _PERIMETER_DCID_FORMAT = "firePerimeter/irwinId/{id}/date/{perimeter_date}"
 _DP1_TOLERANCE = 0.01
-_PERIMETER_URL = "https://services3.arcgis.com/T4QMspbfLg3qTGWY/arcgis/rest/services/Fire_History_Perimeters_Public/FeatureServer/0/query?where=1%3D1&outFields=irwin_IrwinID,irwin_IncidentTypeCategory,poly_DateCurrent,poly_CreateDate&outSR=4326&f=json"
-
+#_PERIMETER_URL = "https://services3.arcgis.com/T4QMspbfLg3qTGWY/arcgis/rest/services/Fire_History_Perimeters_Public/FeatureServer/0/query?where=1%3D1&outFields=irwin_IrwinID,irwin_IncidentTypeCategory,poly_DateCurrent,poly_CreateDate&outSR=4326&f=json"
+_PERIMETER_URL = "https://services3.arcgis.com/T4QMspbfLg3qTGWY/arcgis/rest/services/WFIGS_Interagency_Perimeters/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json"
 _FIRE_INCIDENT_MAP = {
     'CX': 'FireIncidentComplexEvent',
     'WF': 'WildlandFireEvent',
-    'RX': 'PrescribedFireEvent'
+    'RX': 'PrescribedFireEvent',
+    'FA': 'FalseAlarmFireEvent',
+    'MG': 'ManagementFireEvent',
+    'OR': 'OutofAreaResponseFireEvent',
+    'SU': 'IncidentEventSupportEvent',
+    'VF': 'VehicleFireEvent',
+    'SF': 'StructureFireEvent',
+    'DF': 'DebrisProductFireEvent',
+    'PP': 'PreparednessPrepositionFireEvent',
+    'CT': 'ClassroomTrainingFireEvent',
+    'MV': 'MotorVehicleAccidentEvent',
+    'NS': 'NIFC_NSFireEvent'
```

Comment:
Is there some validation results for the changes?

Reply:
Hi Prasanth,

I have added the same  in output CSV into below bucket location[1], their all fire incidents are present on "typeOf" column.

[1] gs://unresolved_mcf/fire/wfigs/perimeter/20240104/perimeter_data.csv.